docs/source/operations/transformations/geogoffset.rst

```diff
@@ -0,0 +1,68 @@
+.. _geogoffset:
+
+================================================================================
+Geographic offsets
+================================================================================
+
+.. versionadded:: 6.0.0
+
+The Geographic offsets transformation adds an offset to the geographic longitude,
+latitude coordinates, and an offset to the ellipsoidal height.
+
++---------------------+----------------------------------------------------------+
+| **Alias**           | geogoffset                                               |
++---------------------+----------------------------------------------------------+
+| **Domain**          | 3D                                                       |
++---------------------+----------------------------------------------------------+
+| **Input type**      | Geodetic coordinates (horizontal), meters (vertical)     |
++---------------------+----------------------------------------------------------+
+| **output type**     | Geodetic coordinates (horizontal), meters (vertical)     |
++---------------------+----------------------------------------------------------+
+
+This method is normally only used when low accuracy is tolerated. It is documented
```

Comment:
>     2\. Don't mention the specific method numbers. They don't add much value to me as a reader and they may get changed upstream which we are not tracking systematically.

I hope they won't !!!! It is quite critical for my current work that the remain stable.

Reply:
> I hope they won't !!!! It is quite critical for my current work that the remain stable.


Do they offer any guarantees for that? I realize that it's not likely to happen, I'm  just wondering if there's something we can do to prevent unpleasant surprises.

Reply:
> Do they offer any guarantees for that? I realize that it's not likely to happen, I'm just wondering if there's something we can do to prevent unpleasant surprises.

My understanding is that they won't reuse numbers. Those are primary keys in a table. They might deprecate entries in favor of new ones. But normally an abandonned method number will not be reused for something different. My WKT -> PROJ converter checks for either the method name (can be a bit fragile too in case of misspellings...) or method code